src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag

```diff
@@ -0,0 +1,81 @@
+                        ${option.content}
+                    </option>
+                </c:forEach>
+            </select>
+        </div>
+    </div>
+    <div class="col-sm-6 padding-0" data-toggle="tooltip" data-placement="top"
+        title="<%= Const.Tooltips.FEEDBACK_SESSION_RECIPIENT %>">
+        <label class="col-sm-5 control-label">
+            Who the feedback is about:
+        </label>
+        <div class="col-sm-7">
+            <select class="form-control participantSelect"
+                id="<%= Const.ParamsNames.FEEDBACK_QUESTION_RECIPIENTTYPE %>${fqForm.questionNumberSuffix}"
+                name="<%= Const.ParamsNames.FEEDBACK_QUESTION_RECIPIENTTYPE %>"
+                <c:if test="${!fqForm.editable}">disabled="disabled"</c:if> onchange="feedbackRecipientUpdateVisibilityOptions(this);getVisibilityMessageIfPreviewIsActive(this);">
+                <c:forEach items="${fqForm.feedbackPathSettings.recipientParticipantOptions}" var="option">
+                    <option <c:forEach items="${option.attributes}" var="attr"> ${attr.key}="${attr.value}"</c:forEach> >
+                        ${option.content}
+                    </option>
+                </c:forEach>
+            </select>
+        </div>
+    </div>
+    <div class="col-sm-6">
+    </div>
+    <div class="col-sm-6 numberOfEntitiesElements${fqForm.question.questionNumber}">
+        <label id="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIES %>_text-${fqForm.question.questionNumber}" class="control-label col-sm-4 small">
+            The maximum number of <span id="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIES %>_text_inner-${fqForm.question.questionNumber}"></span> each respondant should give feedback to:
+        </label>
+        <div class="col-sm-8 form-control-static">
+            <div class="col-sm-6">
+                <input class="nonDestructive" type="radio"
+                    name="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIESTYPE %>"
+                    <c:if test="isNumberOfEntitiesToGiveFeedbackToChecked">checked="checked"</c:if>
+                    value="custom" <c:if test="${!fqForm.editable}">disabled="disabled"</c:if>>
+                <input class="nonDestructive numberOfEntitiesBox" type="number"
+                    name="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIES %>"
+                    id="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIES %>${fqForm.questionNumberSuffix}" 
+                    value="${fqForm.feedbackPathSettings.numOfEntitiesToGiveFeedbackToValue}" 
+                    min="1" max="250" <c:if test="${!fqForm.editable}">disabled="disabled"</c:if>>
+            </div>
+            <div class="col-sm-6">
+                <input class="nonDestructive" type="radio"
+                    name="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIESTYPE %>"
+                    <c:if test="${!isNumberOfEntitiesToGiveFeedbackToChecked}">checked="checked"</c:if>
```

Comment:
The logic related to `isNumberOfEntitiesToGiveFeedbackToChecked` here is wrong.
If `isNumberOfEntitiesToGiveFeedbackToChecked` is true, you'll end up setting the 1st radio button and setting the textbox's value to 1.
Otherwise, you'll end up setting the 2nd radio button (Unlimited) and set the textbox's value to the actual `numberOfEntitiesToGiveFeedbackTo`.

You should reverse the negatives in the `<c:if>` tests here.
Also, the first `<c:if>` is missing the `${}` part.